### ISSUE TITLE
W3C Process & W3C Mission

### ISSUE THREAD

**Issue by fantasai** [w3c/AB-public#220]
The [introduction to the W3C Process](https://www.w3.org/policies/process/#Intro) says:

> W3C work revolves around the standardization of Web technologies. To accomplish this work, W3C follows processes that promote the development of high-quality standards based on the [consensus](https://www.w3.org/policies/process/#Consensus) of the Membership, Team, and public. **W3C processes promote fairness, responsiveness, and progress: all facets of the W3C mission.** This document describes the processes W3C follows in pursuit of its mission.

The quoted aspects of the W3C mission don't seem to be captured by the Vision, and maybe they should be. :)

Relates to https://github.com/w3c/process/issues/555

**Comment by cwilso** [comment]
@fantasai ping on above.

**Comment by fantasai** [comment]
From the perspective of the Process:
- Fairness is about giving participants equal opportunities to participate: not valuing proposals or approvals from one member over another, not valuing one person's opinion over another, not having the chair play favorites but rather encourage the participation of everyone confident and shy... It's about evaluating proposals, statements, suggestions, contributions, and observations on their own merits, not based on who made them. If I make a stupid suggestion in the CSSWG, I expect it to get shot down. If a 12-year-old, or Putin himself, makes a good proposal, I expect it to get adopted (assuming IPR commitments get sorted).
- Responsiveness is about responding to people who engage with us, with use cases, critiques, questions, suggestions; and not ignoring them. (We can be a bit slow here to be honest, since the backlog of issues and requests can be long, but we still try.)
- Progress is about making sure we don't get stuck. The Process for example emphasizes consensus, but it has various escape hatches if you really can't get it: calling for a vote, processing formal objections through the Council, etc. Both the W3C Process and the culture we've developed around implementing it values moving forward and not getting stuck, just as much as it values working really hard to get consensus.

**Comment by cwilso** [comment]
See above on fairness: I think fairness is pretty well covered.

Responsiveness and progress are less directly covered.  I see what you're saying, I think they're heavily related... and I think they're not something we have well-defined principles and value around.  I'm not opposed to developing them, but I think that should be deferred to vNext.

**Comment by fantasai** [comment]
I don't think we do cover fairness, actually. I don't see it in any of our operational principles.

**Comment by cwilso** [comment]
As per [above](https://github.com/w3c/AB-public/issues/220#issuecomment-2518157440):  "the Vision makes multiple statements in order to define "fairness", and enshrine that equity in principles. (User-first, multi-stakeholder/not dominated by ..., diversity, consensus, open participation, etc.). Just saying "be fair" isn't particularly helpful here imo. Can you suggest what's missing?"

**Comment by fantasai** [comment]
As per [above](https://github.com/w3c/AB-public/issues/220#issuecomment-2660581139): "The fact that it doesn't matter who is contributing, what matters is the quality of the contribution, that is something I think needs to be in v1." :)

User-first is about what kinds of proposals we value, not what sources of proposals we value. Multi-stakeholder and open participation and diversity are about who gets to participate. None of these are about whether we tie the value of a proposal to who makes it vs its inherent qualities.
